modules/collector-requirements.adoc

```diff
@@ -12,6 +12,21 @@ It connects to Sensor to report this information.
 To install Collector on systems that have Unified Extensible Firmware Interface (UEFI) and that have Secure Boot enabled, you must use eBPF probes because kernel modules are unsigned, and the UEFI firmware cannot load unsigned packages. Collector identifies Secure Boot status at the start and switches to eBPF probes if required.
 ====
 
+[discrete]
+== Collection requirements
+
+To use the `CORE_BPF` collection method, the base kernel must support BTF, and the BTF file must be available to collector.
```

Comment:
> `and the BTF file must be available to the collector.` 

Perhaps we should be more specific here. This wording implies that something must be done to make the BTF file "available to collector".

Reply:
How about:

```
To use the `CORE_BPF` collection method, the base kernel must support BTF. Collector will search for the BTF symbols in several standard locations.

[ possible list of locations ? ]
```

It might be a bit heavy on implementation detail though - WDYT?

Reply:
> Collector will search for the BTF symbols in several standard locations. [ possible list of locations ? ]

Sounds good to me. It's handled by libbpf, [here](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/tools/lib/bpf/btf.c#n4769) is where it searches for vmlinux btf.